src/platform/datapath_epoll.c

```diff
@@ -1175,7 +1176,7 @@ CxPlatSocketContextUninitialize(
     SocketContext->Uninitialized = TRUE;
 #endif
 
-    if (!SocketContext->IoStarted) {
+    if (!atomic_load_explicit(&SocketContext->IoStarted, memory_order_acquire)) {
```

Comment:
Are you sure we need these? If so, what do you think about making a platform abstraction in quic_platform_posix.h. I suspect this wouldn't be the only place we could/should do this across the code base.

Reply:
Theoretically yes to make read and write before visible in other processor before seeing setting iostarted to true.

Maybe I should add an abstraction and use those __sync_fetch_xxx instead. In the future, if stdatomics is fully supported on msvc, we remove all the abstraction and use the built-in ones that come with c11.